x-pack/plugins/stack_alerts/public/rule_types/es_query/expression/query_form_type_chooser.tsx

```diff
@@ -50,7 +50,7 @@ export const QueryFormTypeChooser: React.FC<QueryFormTypeProps> = ({
   onFormTypeSelect,
 }) => {
   const { uiSettings } = useTriggerUiActionServices();
-  const isEsqlEnabled = uiSettings?.get('discover:enableESQL');
+  const isEsqlEnabled = uiSettings?.get('enableESQL');
```

Comment:
What about using the ENABLE_ESQL constant used also in a previous file for the same code?